index.js

```diff
@@ -1,55 +1,55 @@
 module.exports = {
-  alerts: function alerts() {fetch(`https://api.weather.gov/alerts`).then(res => {return res.json()}).then(data => {console.log(data);}).catch(error => {console.log(error);});},
```

Comment:
Really had to delete and replace all the lines? lmao

Reply:
I used a find and replace, maybe that messed with diff